exercises/armstrong-numbers/metadata.yml

```diff
@@ -0,0 +1,4 @@
+---
+blurb: "Given two decimal parameters (number and base) return True if the number 'number' is an armstrong number with respect to the numeral system with base 'base'."
```

Comment:
Minor nitpick: instead of "return True if", possibly phrase this as "determine whether".

Reply:
"Determine if a number is an Armstrong number" 

would be enough.

The blurb doesn't need to completely specify what you have to do. It should be a teaser to encourage the student to look at the detail of the problem.

Reply:
Ok, I wasn't really sure what the "blurb" was good for ... now i know :smile: